stable/osclass/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: osclass
-version: 5.1.1
+version: 5.1.2
```

Comment:
Now it should be `6.0.1`. A major change (update dependency to a new major version)  was made yesterday